ts/src/pro/woofipro.ts

```diff
@@ -0,0 +1,1160 @@
+// ----------------------------------------------------------------------------
+
+import woofiproRest from '../woofipro.js';
+import { ExchangeError, AuthenticationError } from '../base/errors.js';
+import { ArrayCacheByTimestamp, ArrayCacheBySymbolById, ArrayCache, ArrayCacheBySymbolBySide } from '../base/ws/Cache.js';
+import { Precise } from '../base/Precise.js';
+import { eddsa } from '../base/functions/crypto.js';
+import { ed25519 } from '../static_dependencies/noble-curves/ed25519.js';import type { Int, Str, Strings, OrderBook, Order, Trade, Ticker, Tickers, OHLCV, Balances, Position } from '../base/types.js';
+import Client from '../base/ws/Client.js';
+
+// ----------------------------------------------------------------------------
+
+export default class woofipro extends woofiproRest {
```

Comment:
I think the c# build is broken
![image](https://github.com/ccxt/ccxt/assets/43336371/cfec478d-2490-4f8c-ad2e-3dd3ff71fad5)